### PR TITLE
New minor faction: Colonial minutemen and the unatomized gun sound changes

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
image

Screenshot_2388

The Colonial Minutemen are a group of colonies that have banded together to create a organization which is similar to a police force, and enforces the law on colonies and protects the colonies from threats.
It is said that they found a abandoned warehouse full of blue helmets with 'UN' symbols. Since then, they have adopted the blue helmet's color, and is their official colors now.

NT : heavily dislike
syndicate: neutral
gorelex: hate

New gun sounds and tank sound are from Terragov Marine Corps https://github.com/tgstation/TerraGov-Marine-Corps
Bolt action sounds from CEV Eris https://github.com/discordia-space/CEV-Eris
Loading tank sound from NSV13 https://github.com/BeeStation/NSV13
Blue tank from Haven-Urst https://github.com/Haven-13/Haven-Urist

TODO: write PR
make ship

Some misc changes:
reduces 55.6 and ak47 damage to 30
Demodularizes a bit of files
Makes tanks a bit more balanced
de-shit-i-fys a lot mech code, remind someone to port TG mech to vehicle refactor
Fixes plasma generators from being shit

HUUUUGE FUCKING THANKS TO electricpants#6824 FOR THE M16 SPRITES SERIOUSLY THANK YOU SO MUCH
and @triplezeta and @zestybastard  for spriting help, and @zestybastard for the Minutemen logo
<details><summary>lore</summary>
<p>
https://hackmd.io/d8M4Ejc7Tyut5PkGCLArgQ
</p>
</details>

## Changelog
:cl:
add: new minor faction, colonial minutemen
add: m16, pretty much the same as ak
balance: 556 and ak47 ammo nerfed to 30 damage per hit
soundadd: a ton of gun sounds were changed
refactor: ton of files were demodularize
/:cl:

### CODE REVIEW

**Comment by Valorium-RB** [comment]
Another settleme-I mean space colony needs your help, General

File: code/game/mecha/equipment/weapons/mecha_ammo.dm [hunk 1]
```diff
@@ -75,6 +80,25 @@
...
+	load_audio = 'sound/weapons/gun/general/load_shell.ogg'
+	ammo_type = "missiles_he"
+
+/obj/item/mecha_ammo/tank_shell/update_name() // i hate the fact that i think this is the only way to do this, what the fuck
+	if(!rounds)
+		qdel(src)
+
+/obj/item/mecha_ammo/tank_shell/Initialize()
+	. = ..()
+	AddComponent(/datum/component/two_handed, require_twohands=TRUE)
```
Comment (Kapu1178) [inline code/game/mecha/equipment/weapons/mecha_ammo.dm]: Use /ComponentInitialize() not /Initialize() for this

File: code/game/mecha/equipment/tools/other_tools.dm [hunk 2]
```diff
@@ -360,7 +360,7 @@
 
 /obj/item/mecha_parts/mecha_equipment/generator
 	name = "exosuit plasma converter"
-	desc = "An exosuit module that generates power using solid plasma as fuel. Pollutes the environment."
+	desc = "An exosuit module that generates power using solid plasma as fuel."
```
Comment (Kapu1178) [inline code/game/mecha/equipment/tools/other_tools.dm]: keep the funny comment

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 3]
```diff
@@ -212,12 +212,18 @@
...
 	var/projectiles_cache //ammo to be loaded in, if possible.
 	var/projectiles_cache_max
 	var/projectile_energy_cost
 	var/disabledreload //For weapons with no cache (like the rockets) which are reloaded by hand
 	var/ammo_type
 
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/Initialize() //initial(projectiles) prevented me from making mech weapons start empty TODO: PORT ALL OF TG MECH IMPROVEMENTS
+	. = ..()
+	if(!projectiles_max)
+		projectiles_max = projectiles
```
Comment (Kapu1178) [inline code/game/mecha/equipment/weapons/weapons.dm]:
```suggestion
	projectiles_max ||= projectiles
```

**Review by Kapu1178 — DISMISSED** [review]

**Comment by 1glitchycent** [comment]
man I swear I've seen this before

**Comment by rye-rice** [comment]
This should be ready, leaving map for later

**Comment by Kapu1178** [comment]
ill review this later this week

**Review by triplezeta — APPROVED** [review]
sprites good

File: code/game/mecha/combat/five_stars.dm [hunk 4]
```diff
@@ -1,22 +1,41 @@
...
+	ME = new /obj/item/mecha_parts/mecha_equipment/generator(src)
 	ME.attach(src)
+
+/obj/mecha/combat/combat_tank/loaded/Initialize()
+	. = ..()
+	max_ammo()
+
+/obj/mecha/combat/combat_tank/minutemen
+	name = "\improper Type 325 Armored Combat Vehicle \"Blueberry\""
+	desc = "A heavily modified Type 207 brought up to modren standards. Usually, third-parties can't afford to purchase exosuits, so they have to rely on refitting old 207s."
```
Comment (MarkSuckerberg) [inline code/game/mecha/combat/five_stars.dm]:
```suggestion
	desc = "A heavily modified Type 207 brought up to modern standards. Usually, third-parties can't afford to purchase exosuits, so they have to rely on refitting old 207s."
```

File: code/game/mecha/combat/five_stars.dm [hunk 5]
```diff
@@ -1,22 +1,41 @@
-/obj/mecha/combat/five_stars
-	desc = "A state of the art tank deployed by the Spinward Stellar Coalition National Guard."
-	name = "\improper Tank"
+/obj/mecha/combat/combat_tank
+	name = "\improper  Armored Combat Vehicle \"Tank\""
+	desc = "While exosuits have taken over armored vehicles in peformance in ground combat, poorer factions have to rely on these old relics. The 207 was exclusively mass produced for a Sarthini war, before a abrupt surrender, leaving many of these to gain dust in warehouses."
```
Comment (MarkSuckerberg) [inline code/game/mecha/combat/five_stars.dm]: Sarthini?

File: code/game/mecha/combat/five_stars.dm [hunk 6]
```diff
@@ -1,22 +1,41 @@
-/obj/mecha/combat/five_stars
-	desc = "A state of the art tank deployed by the Spinward Stellar Coalition National Guard."
-	name = "\improper Tank"
+/obj/mecha/combat/combat_tank
+	name = "\improper  Armored Combat Vehicle \"Tank\""
```
Comment (MarkSuckerberg) [inline code/game/mecha/combat/five_stars.dm]:
```suggestion
	name = "\improper Armored Combat Vehicle \"Tank\""
```

File: code/game/objects/items/storage/belt.dm [hunk 7]
```diff
@@ -395,7 +395,7 @@
 
 /obj/item/storage/belt/military
 	name = "chest rig"
-	desc = "A set of tactical webbing worn by Syndicate boarding parties."
+	desc = "A set of tactical webbing worn by military cosplayers and actual militaries."
```
Comment (MarkSuckerberg) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	desc = "A set of tactical webbing worn by military cosplayers and actual militaries alike."
```

File: code/game/objects/structures/signs/_signs.dm [hunk 8]
```diff
@@ -250,6 +250,12 @@
 /obj/structure/sign/solgov_flag/right
 	icon_state = "solgovflag-right"
 
+// colonial minutemen seal
+/obj/structure/sign/minutemen
+	name = "Banner of the Colonial Minutemen"
+	desc = "A seal representing the many colonies of that comprise of the Colonial Minutemen."
```
Comment (MarkSuckerberg) [inline code/game/objects/structures/signs/_signs.dm]:
```suggestion
	desc = "A seal representing the many colonies comprising the Colonial Minutemen."
```

File: code/modules/clothing/under/jobs/security.dm [hunk 9]
```diff
@@ -251,6 +193,41 @@
...
+	name = "constable outfit"
+	desc = "A british looking outfit."
+	icon_state = "constable"
+	item_state = "constable"
+	can_adjust = FALSE
+	custom_price = 200
+
+/obj/item/clothing/under/rank/security/officer/spacepol
+	name = "police uniform"
+	desc = "Space not controlled by megacorporations, planets, or pirates is under the jurisdiction of Spacepol."
```
Comment (MarkSuckerberg) [inline code/modules/clothing/under/jobs/security.dm]:
```suggestion
	desc = "A spiffy uniform poking fun at the lack of law enforcement on the frontier sector."
```
(or literally just any description besides the current one)

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

File: code/game/mecha/combat/five_stars.dm (hunk 5 above)
Comment (rye-rice) [reply code/game/mecha/combat/five_stars.dm]: desc = "While exosuits have taken over armored vehicles in peformance in ground combat, factions that can not afford expensive exosuits have to rely on these old relics. The 207 was exclusively mass produced for a Kalixcis war, however a abrupt surrender left many of these to gain dust in warehouses."
is this better?

Comment (triplezeta) [reply code/game/mecha/combat/five_stars.dm]: Kalixcian?

Comment (triplezeta) [reply code/game/mecha/combat/five_stars.dm]: an abrupt surrender

Comment (rye-rice) [reply code/game/mecha/combat/five_stars.dm]: desc = "While exosuits have taken over armored vehicles in performance in ground combat, factions that can not afford expensive exosuits have to rely on these old relics. The 207 was exclusively mass produced for a Kalixcian war, however an abrupt surrender left many of these to gain dust in warehouses."

File: whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 10]
```diff
@@ -63,8 +59,23 @@
...
 /obj/item/ammo_box/magazine/ebr
 	name = "M514 EBR Magazine (.308 Winchester FMJ)"
-	icon = 'whitesands/icons/obj/ammo.dmi'
 	icon_state = "ebr_mag"
 	ammo_type = /obj/item/ammo_casing/ballistic/win308
 	caliber = ".308 Winchester FMJ"
 	max_ammo = 10
+
+/obj/item/ammo_box/magazine/p16
+	name = "P-16 Magazine (7.62x39mm FMJ)"
```
Comment (Apogee-dev) [inline whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm]: name = "P-16 Magazine (5.56mm)"

File: whitesands/code/modules/projectiles/guns/ballistic/automatic.dm [hunk 11]
```diff
@@ -223,3 +223,23 @@
 	righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
 	mob_overlay_icon = 'icons/mob/clothing/back.dmi'
 	mag_display = TRUE
+
+/obj/item/gun/ballistic/automatic/p16
+	name = "P-16"
+	desc = "A old rifle used by professional mercanaries. It is said that the P-16 and AK-47 destined to be in combat against each other. Chambered in 5.56mm."
```
Comment (Apogee-dev) [inline whitesands/code/modules/projectiles/guns/ballistic/automatic.dm]: desc = "An ancient rifle used by professional mercenaries. It is said that the P-16 and AK-47 were destined to be in combat against each other. Chambered in 5.56mm."

File: whitesands/code/modules/projectiles/guns/ballistic/automatic.dm [hunk 12]
```diff
@@ -223,3 +223,23 @@
...
+	fire_rate = 5
+	mag_display = TRUE
+	weapon_weight = WEAPON_MEDIUM
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = ITEM_SLOT_BACK
+	mag_type = /obj/item/ammo_box/magazine/p16
+
+/obj/item/gun/ballistic/automatic/p16/minutemen
+	name = "CM-16"
+	desc = "A heavily modified version of the P-16. Standard rifle of the Colonial Minutemen"
```
Comment (Apogee-dev) [inline whitesands/code/modules/projectiles/guns/ballistic/automatic.dm]: desc = "A heavily modified version of the P-16. Standard rifle of the Colonial Minutemen."

**Review by Apogee-dev — CHANGES_REQUESTED** [review]

**Comment by rye-rice** [comment]
thank you apogee!

**Comment by Zevotech** [comment]
you mean i get to be a spacecop AND hate solgov and NT? amazing! how do i join

**Comment by Ms-Mee** [comment]
> you mean i get to be a spacecop AND hate solgov and NT? amazing! how do i join

First you must give away all your greytide and give into order, that is your purpose, keep the colonies safe and organized, free from the grasp of meddling corporations and outside governmental forces

**Review by MarkSuckerberg — APPROVED** [review]

**Comment by rye-rice** [comment]
The lore was rewritten, but there's no need to touch the code, should still be ready to merge

**Pull request merged**